src/app/widgets/climate.cpp

```diff
@@ -80,6 +79,8 @@ void ClimateSnackBar::set_max_fan_speed(int max_speed)
 
 void ClimateSnackBar::set_fan_speed(int speed)
 {
+    this->repaint();
```

Comment:
the repaint shouldnt be needed anymore right?

i moved that into the ClimateState widget so the entire popup doesn't need to be repainted

Reply:
😅 this is what taking month long gaps does to me - I'll remove this, it's a remnant of debugging